practice/jokes/jokes/static/jokes/site.js

```diff
@@ -0,0 +1,12 @@
+'use strict';
+
+var setups = $('.setup');
+
+/**
+ * Add click handlers to each of the setups
+ */
+ var click_handler = setups.on('click', function() {
+   return $(this).next().show()
+ });
+
+setups.append(click_handler)
```

Comment:
I don't think this last line does anything. Calling `.append()` on an element section adds a new inner element. I don't know what it does when you add a list of click handlers?